verispy/__init__.py

```diff
@@ -5,4 +5,4 @@
 
 from .veris import VERIS
 
-__version__ = '0.1.9'
\ No newline at end of file
+__version__ = '0.1.10'
```

Comment:
also need to update setup.py with the version

Reply:
Fixed in 04e2c71.